src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[];
+};
+// eslint-disable-next-line
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  return (
+    <div className="CommentList">
+      {comments.map(coment => (
+        <CommentInfo {...coment} key={coment.id} />
```

Comment:
spread props it's not a good idea in this case. it's better to pass certain logical object as separated prop
```suggestion
        <CommentInfo comment={comment} key={coment.id} />
```